src/Experiment.js

```diff
@@ -3,7 +3,8 @@ import OptimizeContext from "./OptimizeContext";
 
 class Experiment extends React.Component {
   static defaultProps = {
-    loader: null
+    loader: null,
+    timeout: 3000
```

Comment:
Could this be configurable? It feels a bit too long.

Reply:
Yes, it's a prop so just pass whatever you want to the `<Experiment ... />`

If you want to configure it globally you could `Experiment.defaultProps.timeout = 1000` during app startup.